wasm/src/debug.rs

```diff
@@ -1,5 +1,7 @@
 //! WASM debugging utilities
 
+#![allow(clippy::single_match_else, clippy::option_if_let_else)]
```

Comment:
would you rather not apply suggestions?

Reply:
Yes, I think it makes code in this file look uglier. But I can rewrite it if you want

Reply:
I don't really care because I don't see much value in this lints anyhow. However, since we have them I always try to apply them and avoid the stress. If you don't want to apply them then I don't think it's a good idea to override them at the module level. Rather you should do it on a per case basis because I can't see which piece of code are these `allows` referring to and I can't make an evaluation of your judgement

Our policy around lints is very loose